content/source/docs/enterprise/workspaces/run-basics.html.md

```diff
@@ -26,17 +26,79 @@ When you request a run, TFE locks the run to the current Terraform code (usually
 
 ## Run States
 
-A given Terraform run can be in the following states:
+Each run passes through several stages of action, and TFE shows the progress through those stages as run states.
+
+In the list of workspaces on TFE's main page, each workspace shows the state of its most recently queued run. So if any runs are pending, the workspace is in "pending," regardless of what it's currently running or waiting for.
```

Comment:
This is wrong. We are using current run now.

Reply:
Ah, thank you. I'll fix.